### DISCUSSION TITLE
Create playlist from Genre selection ?

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1449] [Fixed Issues] [answered]
I am sure I used to be able to do this but either can't remember how or the functionality has disappeared.  I would like to be able to select all (or some) albums from a Genre, add them to a playlist and may be randomise the playlist.

Can anybody jog my memory - is there a way to do this please ?

Thanks

Chris

**Comment by ceejayemm** [comment]
Anybody ?

**Comment by ceejayemm** [comment]
@rern 
Nobody else seems to know how/if this can be done. Can you comment ?

Thanks

Chris

**Reply by rern** [reply, marked as answer]
It's a bug of context menu on non-file modes.
Please try the UPDATE branch.

**Reply by ceejayemm** [reply]
@rern 
Many Thanks, that seems to solve the problem.

Chris

**Comment by ceejayemm** [comment]
Update branch resolves this problem
